hera_cal/abscal_funcs.py

```diff
@@ -1309,7 +1309,7 @@ def interp2d_vis(model, model_lsts, model_freqs, data_lsts, data_freqs, flags=No
     return DataContainer(new_model), DataContainer(new_flags)
 
 
-def rephase_vis(model, model_lsts, data_lsts, bls, freqs, flags=None, max_dlst=0.005, latitude=-30.72152):
+def rephase_vis(model, model_lsts, data_lsts, bls, freqs, inplace=False, flags=None, max_dlst=0.005, latitude=-30.72152):
```

Comment:
I think this function was originally `inplace` by default b/c this is required by the lstbinner. can we keep this default to True and make wherever abscal calls it feed it True?

Reply:
To be clear, the function it calls (in hera_cal.utils) has default inplace=True. This function, in abscal_funcs, didn't expose that parameter until now.

I hesitate to change the default (which was effectively False before because it was hard-coded to be False) unless you think it's OK (it's your code after all).

Reply:
Anyway, I'm gonna merge it in and if you don't like the default, we can change it later.